operator/controllers/routing_table.go

```diff
@@ -58,12 +54,44 @@ func addAndUpdateRoutingTable(
 			host,
 		)
 	}
-	return pingInterceptors(
-		ctx,
-		cl,
-		http.DefaultClient,
-		namespace,
-		interceptorSvcName,
-		interceptorSvcPort,
-	)
+	return updateRoutingMap(ctx, cl, namespace, table, lggr)
+}
+
+func updateRoutingMap(
+	ctx context.Context,
+	cl client.Client,
+	namespace string,
+	table *routing.Table,
+	lggr logr.Logger,
```

Comment:
let's put the `lggr` parameter after `ctx`, to be consistent

```suggestion
	ctx context.Context,
	lggr logr.Logger,
	cl client.Client,
	namespace string,
	table *routing.Table,
```